burgerpy/modules/qpick.py

```diff
@@ -0,0 +1,39 @@
+from hashlib import sha1
+import urllib2
+import simplejson as json
+from burgerpy.common import Module, Config
+
+# picks order according to quantum RNG --- entangles user with a photon in Australia, basically.
```

Comment:
is there a way to get a photon from country that's not part of the Five Eyes program? I don't trust that the photon won't be cryptographically compromised, basically.